src/libostree/ostree-repo-commit.c

```diff
@@ -655,9 +792,6 @@ write_object (OstreeRepo         *self,
           gs_unref_object GConverter *zlib_compressor = NULL;
           gs_unref_object GOutputStream *compressed_out_stream = NULL;
 
-          if (self->generate_sizes)
-            indexable = TRUE;
```

Comment:
This change isn't really explained in any of the commit messages. It seems to me there are 2 implications of it:

1. Sizes will be generated for not only file objects, but also metadata objects.
2. Sizes will be generated for all repository types, not just archive-z2.

Is that correct? Are there any other implications of this change?